src/vs/platform/quickinput/browser/quickInputList.ts

```diff
@@ -543,6 +544,30 @@ class QuickPickItemElementRenderer extends BaseQuickInputListRenderer<QuickPickI
 		} else {
 			data.entry.classList.remove('has-actions');
 		}
+		// --- Command Palette: show command description as native tooltip ---
+		const commandId =
+			(element.item as any)?.command ??
+			(element.item as any)?.commandId ??
+			(element.item as any)?.id;
+
+		if (typeof commandId === 'string') {
+			const command = CommandsRegistry.getCommand(commandId);
+
+			let tooltip: string | undefined;
+
+			if (command?.metadata?.description) {
+				if (typeof command.metadata.description === 'string') {
+					tooltip = command.metadata.description;
+				} else {
+					tooltip = command.metadata.description.value;
+				}
+			}
+
+			if (tooltip) {
+				data.entry.title = tooltip;
+			}
+		}
+
 	}
```

Comment:
This change is changing the quick pick component for _all_ quick picks. I would expect a change only for the command palette, which is called the "Commands Quick Access". 

https://github.com/microsoft/vscode/blob/8e8e866dc462652b87189eacd85b13402e033bcb/src/vs/platform/quickinput/browser/commandsQuickAccess.ts#L49